english/candyland/obamacare/signup.md

```diff
@@ -7,3 +7,5 @@ The system was always fixed - the little guy always gets screwed.
 But now's your chance to get back at the man!  Free Health Care!  Paid for by...oh, the little guy again.
 
 Go ahead, eat all the marshmallow stuff, it's too late anyway.
+
+[Resign thyself](english/candyland/obamacare/hillary/hillary-appears.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.